__tests__/app.test.js

```diff
@@ -61,4 +61,64 @@ describe('GET api/articles', () => {
               });
         })
     })
+})
+describe('GET /api/articles/:article_id', () => {
+    test('200: response with an article objects by provided articel id with properties: author, title, article_id, body, topic, created_at, votes', () => {
```

Comment:
Be a little careful with spelling and grammar in you test descriptions. 
it will respond with a single article object - but this description says 'objects' which suggests plural.

Reply:
done